PackageExplorer/PackageChooser/PackageDetailActionsControl.xaml.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+using NuGetPe;
+using PackageExplorerViewModel;
+
+namespace PackageExplorer
+{
+    /// <summary>
+    /// Interaction logic for PackageRowDetails.xaml
```

Comment:
`PackageDetailActionsControl.xaml`

Reply:
@campersau Ah, this is left over from my renaming the control 😄 Will update.